l10n_ch_qr_bill_scan/wizard/account_invoice_import.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2020 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.modules.module import get_resource_path
+
+from ..tools import QR
+
+logger = logging.getLogger(__name__)
+
+try:
+    import cv2
+    import numpy as np
+    import pdf2image
+    from pyzbar.pyzbar import ZBarSymbol, decode
+except ImportError:
+    # Necessary libraries to decode QR from pdf are
+    # not installed
+    logger.warning(
```

Comment:
Maybe we should also log the original error to help troobleshooting (in my case some missing system packages such as python3-opencv libzbar0 poppler-utils